views/stylesheet.css

```diff
@@ -0,0 +1,59 @@
+.calculator-container {
+  width: 232px;
+  height: 323px;
+  border-radius: 15px;
+  position: absolute;
```

Comment:
I believe because I have the size of the calculator constricted to specific and explicit dimensions that this prevents the calculator from stretching to the size of the screen. Also, when I remove the margin : auto, the calculator is no longer centered.  Actually removing any of the positioning styles removes the calculator from the center of the page